reference/checkouts.v3.yml

```diff
@@ -2114,101 +2268,251 @@ definitions:
+                    type: object
+                    properties:
+                      name:
+                        description: ''
+                        type: string
+                      message:
+                        description: ''
+                        type: string
+                      amount:
+                        description: ''
+                        type: number
+                        format: double
+                      amount_as_integer:
+                        description: ''
+                        type: number
+                        format: double 
+                    required:
+                    - quantity
+          required:
+            - physical_items
+            - digital_items
+            - gift_certificates
+      created_time:
+        description: Time when the cart was created.
+        type: string
+      updated_time:
+        description: Time when the cart was last updated.
+        type: string
+  Currency:
+    title: Currency
+    type: object
+    properties:
+      code:
+        description: 'ISO-4217 currency code. (See: http://en.wikipedia.org/wiki/ISO_4217.)'
+        type: string
+  AppliedCoupon:
+    title: Applied Coupon
+    type: object
+    properties:
+      code:
+        description: the coupon code
+        type: string
+      id:
+        description: The coupon ID.
+        type: string
+      discounted_amount:
+        description: The discounted amount applied within a given context.
+        type: number
+        format: double
+      coupon_type:
+        type: integer
+        enum:
+          - 1
+          - 2
+          - 3
+          - 4
+          - 5
+        description: |-
+          |Type `int`|Type Name|
+          |-|-|
+          |`0`|`per_item_discount`|
+          |`1`|`percentage_discount`|
+          |`2`|`per_total_discount`|
+          |`3`|`shipping_discount`|
+          |`4`|`free_shipping`|
+          |`5`|`promotion`|
+    required:
+      - code
+  AppliedDiscount:
+    title: Applied Discount
+    type: object
+    properties:
+      id:
+        description: ID of the applied discount.
+        type: number
+        format: double
+      discounted_amount:
+        description: The discounted amount applied within a given context.
+        type: number
+        format: double
+  LineItem:
+    title: Line Item
+    type: object
+    properties:
+      physical_items:
+        description: ''
+        type: array
+        items:
+          title: Item Physical
+          type: object
+          properties:
+            quantity:
+              description: ''
+              type: number
+              format: double
+            id:
+              description: The line-item ID.
+              type: string
+            variant_id:
+              description: ''
+              type: number
+              format: double
+            product_id:
+              description: ''
+              type: number
+              format: double
```

Comment:
I think the format needs to be updated, unless product_id can have fractional values.

Reply:
removed